services/libs/tinybird/pipes/insightsProjects_filtered.pipe

```diff
@@ -1,6 +1,10 @@
 DESCRIPTION >
     Provides filters for projects. Merges collection slug from associated collections. Merges segment aggregates from segmentsAggregatedMV
 
+TOKEN "insighsProjects_filtered_endpoint_read_2583" READ
+TOKEN "raul_dev_access_token" READ
+TOKEN "insights-app-token" READ
+
```

Comment:
can you also remove these? shouldn't be needed. Would be great if u can push the pipe definition back to staging without these so next pull won't generate it